src/index.ts

```diff
@@ -180,34 +195,87 @@ export class Snackbar {
     // Restart timer when mouseleave
     el.addEventListener('mouseenter', () => {
       this.stopTimer()
+      this.expand()
     })
     el.addEventListener('mouseleave', () => {
       this.startTimer()
+      this.stack()
     })
 
     this.el = el
 
     this.wrapper.appendChild(el)
   }
 
+  stack() {
+    instanceStackStatus[this.options.position] = true
+    const positionInstances = instances[this.options.position]
+    const l = positionInstances.length - 1
+    positionInstances.forEach((instance, i) => {
+      if (instance.el) {
+        instance.el.style.transform = `translate3d(0, -${(l - i) * 15}px, -${l -
+          i}px) scale(${1 - 0.05 * (l - i)})`
+        if (l - i >= 3) {
+          instance.el.style.opacity = '0'
+        } else {
+          instance.el.style.opacity = '1'
+        }
+      }
+    })
+  }
+
+  expand() {
```

Comment:
It should not self-destroy when expanded

Reply:
I'm confused. There is no destroy logic in `expand`

Reply:
The snackbar will self-destroy when `timeout` option is set.

Reply:
Ah. thanks for the heads up!